fuzz/fuzz_pcap.c

```diff
@@ -0,0 +1,115 @@
+#include <config.h>
+#include "netdissect-stdinc.h"
+#include <sys/stat.h>
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <fcntl.h>
+#include <errno.h>
+#include <string.h>
+
+#include <pcap/pcap.h>
+#include "netdissect.h"
+#include "print.h"
+
+FILE * outfile = NULL;
+
+static int
+fuzz_ndo_printf(netdissect_options *ndo, const char *fmt, ...)
+{
+    va_list args;
+    int ret;
+
+    va_start(args, fmt);
+    ret = vfprintf(outfile, fmt, args);
+    va_end(args);
+
+    return (ret);
+}
+
+void fuzz_openFile(const char * name) {
+    if (outfile != NULL) {
+        fclose(outfile);
+    }
+    outfile = fopen(name, "w");
+}
+
+static int bufferToFile(const char * name, const uint8_t *Data, size_t Size) {
+    FILE * fd;
+    if (remove(name) != 0) {
+        if (errno != ENOENT) {
+            printf("failed remove, errno=%d\n", errno);
+            return -1;
+        }
+    }
+    fd = fopen(name, "wb");
+    if (fd == NULL) {
+        printf("failed open, errno=%d\n", errno);
+        return -2;
+    }
+    if (fwrite (Data, 1, Size, fd) != Size) {
+        fclose(fd);
+        return -3;
+    }
+    fclose(fd);
+    return 0;
+}
+
+int LLVMFuzzerTestOneInput(const uint8_t *Data, size_t Size) {
+    pcap_t * pkts;
+    char errbuf[PCAP_ERRBUF_SIZE];
+    const u_char *pkt;
+    struct pcap_pkthdr *header;
+    int r;
+    u_int packets_captured = 0;
+    netdissect_options Ndo;
+
+    //initialize output file
```

Comment:
The libfuzzer documentation https://llvm.org/docs/LibFuzzer.html#id38 states that 
> Do *this* only if you really need to access argv/argc.

So, why do you remand the use of `LLVMFuzzerInitialize` ?

Reply:
I hadn't seen that documentation. I don't understand that statement (because why is it better to use a static local variable rather than an explicit initialize-time function call?), but, ok, that's what they want.

Reply:
One reason that I want to use the Initialize function is the way I structured my individual fuzzers; the common code in common.c has the common initialization.

The alternative is to add code like "if (!initialized) { intialize(); initialized=1; }" to each of the individual fuzzers, but using the LLVMFuzzerInitialize function means that I don't have to duplicate boilerplate code like that.

Reply:
I do not know why they prefer it this way.
Another alternative could be to use the static `initialized` variable in `LLVMFuzzerInitialize`